vital/types/geo_point.cxx

```diff
@@ -136,10 +137,10 @@ operator<<( std::ostream& str, vital::geo_point const& obj )
     auto const old_prec = str.precision();
     auto const loc = obj.location();
 
-    str << std::setprecision(22)
+    str << std::setprecision(std::numeric_limits<double>::digits10 + 2)
         << "[ " << loc[0]
-        << ", " << loc[1]
-        << ", " << loc[2]
+        << " , " << loc[1]
```

Comment:
geo_polygon prints `" / "` between coordinates, so I figured this would be an OK change. It may make parsing a bit easier for others as well, but I'm happy to change it back

Reply:
... but "/" usually *does* have whitespace on either (or neither) side, whereas "," usually has trailing space with no preceding space. Partly, it comes down to whether this is meant to be read by humans or machine, and my vague impression is more toward the former. [Other](https://github.com/Kitware/kwiver/blob/f75454864fab5a5a7b3b580ae9730a8b4ad7d7b6/vital/types/point.cxx#L47-L91) [places](https://github.com/Kitware/kwiver/blob/f75454864fab5a5a7b3b580ae9730a8b4ad7d7b6/vital/types/timestamp.cxx#L204-L258) also use "," with no preceding space, although most `operator<<`s seem to not use "," at all. (Incidentally, `geo_polygon` using "/" appears to be a concession to that being a list of tuples, since the *tuples* are separated by "," with ordinary English spacing.)

Reply:
...If parsing is the main concern, I would probably just drop the "," entirely...

Reply:
I think you're right in that it is primarily meant to be human readable, so we may want to stick with the comma in ordinary English spacing, then. Does keeping it make sense to you, or would you prefer it be changed anyway?